frontend/src/component/integrations/IntegrationList/AvailableIntegrations/AvailableIntegrations.tsx

```diff
@@ -225,9 +225,9 @@ export const AvailableIntegrations: VFC<IAvailableIntegrationsProps> = ({
                                     target='_blank'
                                     rel='noopener noreferrer'
                                 >
-                                    Unleash front-end API
+                                    Unleash Frontend API
                                 </a>
-                                , but not to the regular Unleash client API.
+                                , but not to the regular Unleash Client API.
```

Comment:
I'm tempted to call this the backend API instead (vs. frontend API), but you probably already considered it and decided to keep it as client API for now.

Reply:
Agreed. But it almost makes me want to add a `/backend` endpoint that does the same thing as the client API 😄